Department-for-Transport-Disruptions/create-disruptions-data
# Add display id to disruptions to uniquely identify disruptions

Testing instructions

- Create a new disruption or edit an existing disruption should create a display id. This is visible when you export data in view-all-disruptions page or visible in dashboards page
